### ISSUE TITLE
Revisit L1 pinging to check if the destination is live

### ISSUE THREAD

**Issue by cubranic** [kotekan/kotekan#650]
In #527 I added detection of missing/dead L1 nodes by periodic pinging. This was turned off in #634 because during deployment we ran into frequent cases of masses of GPU nodes declaring multiple destinations "dead" because they had not responded to pings for too long:

```
Dec 01 17:40:49 cn3g1 kotekan[14273]: /frb/network_send: Too long since last ping response (06:55.551), mark host 10.9.204.18 dead.
Dec 01 17:40:48 cn3g1 kotekan[14273]: /frb/network_send: Too long since last ping response (06:31.682), mark host 10.8.206.15 dead.
Dec 01 17:40:47 cn3g1 kotekan[14273]: /frb/network_send: Too long since last ping response (06:31.681), mark host 10.8.202.11 dead.
Dec 01 17:40:44 cn3g1 kotekan[14273]: /frb/network_send: Too long since last ping response (06:30.909), mark host 10.8.206.19 dead.
Dec 01 17:40:41 cn3g1 kotekan[14273]: /frb/network_send: Too long since last ping response (06:31.017), mark host 10.9.202.19 dead.
Dec 01 17:40:39 cn3g1 kotekan[14273]: /frb/network_send: Too long since last ping response (06:30.659), mark host 10.7.207.18 dead.
Dec 01 17:40:38 cn3g1 kotekan[14273]: /frb/network_send: Too long since last ping response (06:30.263), mark host 10.7.209.17 dead.
Dec 01 17:40:37 cn3g1 kotekan[14273]: /frb/network_send: Too long since last ping response (06:30.050), mark host 10.7.205.16 dead.
Dec 01 17:40:36 cn3g1 kotekan[14273]: /frb/network_send: Too long since last ping response (06:53.299), mark host 10.8.210.19 dead.
Dec 01 17:40:33 cn3g1 kotekan[14273]: /frb/network_send: Too long since last ping response (06:51.049), mark host 10.7.203.11 dead.
Dec 01 17:40:31 cn3g1 kotekan[14273]: /frb/network_send: Too long since last ping response (06:49.295), mark host 10.7.203.19 dead.
Dec 01 17:40:25 cn3g1 kotekan[14273]: /frb/network_send: Too long since last ping response (06:46.014), mark host 10.8.208.10 dead.
Dec 01 17:40:23 cn3g1 kotekan[14273]: /frb/network_send: Too long since last ping response (06:45.385), mark host 10.7.203.14 dead.
Dec 01 17:40:22 cn3g1 kotekan[14273]: /frb/network_send: Too long since last ping response (06:44.364), mark host 10.6.203.12 dead.
Dec 01 17:40:15 cn3g1 kotekan[14273]: /frb/network_send: Too long since last ping response (06:40.202), mark host 10.7.201.15 dead.
Dec 01 17:40:10 cn3g1 kotekan[14273]: /frb/network_send: Too long since last ping response (06:50.367), mark host 10.9.208.18 dead.
```

This needs a more detailed check to see whether it happened because it would be a good feature to have.

Log snapshot: [kotekan-log-650.gz](https://github.com/kotekan/kotekan/files/3953298/kotekan-log-650.gz)

**Comment by cubranic** [comment]
It is entirely possible that if too many nodes are in the quick-pinging mode, because the ping-receive loop is single threaded, with up to a second wait for the response, checks bunch up and we just run out of time to process them all, so some of them end up being over 30s without a response and are marked dead.

Instead, we should probably have two threads, one for sending pings and a separate one for receiving.

**Comment by andrerenard** [comment]
This was closed by #703
